client/src/App.jsx

```diff
@@ -0,0 +1,75 @@
+import React, { setState, useState, useEffect } from 'react';
+import axios from 'axios';
+import Averages from './Averages.jsx'
+import ReviewBlurb from './ReviewBlurb.jsx'
+import Modal from './Modal.jsx'
+
+const App = () => {
+  const [averages, setAverages] = useState('')
```

Comment:
maybe in class tomorrow you can show me how all this is working in action and we can find a way of cleaning it up a bit as well as figure out what is going on with the hook error